src/lib.ts

```diff
@@ -115,6 +115,7 @@ export type BonusEquipMode = "free" | "embezzler" | "dmt" | "barf";
 export const WISH_VALUE = 50000;
 export const HIGHLIGHT = isDarkMode() ? "yellow" : "blue";
 export const ESTIMATED_OVERDRUNK_TURNS = 60;
+export const EMBEZZLER_MULTIPLIER = 2.5;
```

Comment:
my only small note -- any appetite to make this toggleable by a runtime flag or a pref? only reason i ask -- it makes testing different values over a long period of time a little bit easier and i think it would be interesting to run garbo for a week while doing a walk from 1.5 to 3 by increments of 0.2 and seeing where the profitability sweet spot ends up if you take out outlier items and such. it is ok if the answer is no, i know we hate prefs, this just seems like a neat testing one.

Reply:
To be honest this should be estimateable by creating mock embezzler and barf outfits, and then comparing their meat drop% vs their bonus drops, and also comparing the meat mood for embezzlers vs meat mood for barf.

I just haven't figured out a good way to create these mock outfits to compare.  Using the 5 parameter `maximize` with `include_equip`  will give us a record that includes the equipment it wants to use, but I haven't worked much with this to really know for sure how realistic this is as a solution.